lib/ffigen.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// A bindings generator for dart.
+///
+/// See complete usage at - https://pub.dev/packages/ffigen.
+library ffigen;
+
+export 'src/code_generator.dart';
```

Comment:
You might want to be particular about what you allow to export from these three files. Once people start using this, you cannot change it anymore.

Reply:
Ah, right.
Updated to expose just the things required to build your own `bin/ffigen.dart` script. If the user wants even more flexibility they can always import from `lib/src`.